documentation/plain_rules.md

```diff
@@ -173,31 +182,67 @@ It could happen (in theory) that a just got auth token also produce a 401 Not au
 
 
 ### HTTP POST action
-Makes an HTTP POST to an URL specified in `url` inside `parameters`, sending a body built from `template`.
+Makes an HTTP POST to an URL specified in `url` inside `parameters`, sending a body built from `template`. 
+The `parameters` field can specify
+* method: *optional*, HTTP method to use, POST by default
+* **url**: *mandatory*, URL target of the HTTP method
+* headers: *optional*, an object with fields and values for the HTTP header
+* qs: *optional*, an object with fields and values to build the query string of the URL
+* json: *optional*, an object that will be sent as JSON. String substitution will be performed in the keys and 
+values of the object's fields, but only the form `{{X}}` is allowed. If present, it overrides `template` from `action` 
+
+```json
+ "action":{
+      "type":"post",
+      "template":"BloodPressure is ${BloodPressure}",
+      "parameters":{
+         "url": "http://localhost:9182/${type}/${id}",
+         "method": "PUT",
+         "headers": {
+            "Content-type": "text/plain",
+            "X-BloodPressure": "${BloodPressure}"
```

Comment:
It would be interesting to show a case in which `${}` is used for a key in either `headers` or `qs`. Otherwise, the reader could wrongly think that macros can be only used in the value part of the JSONs.